graspologic/inference/__init__.py

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) Microsoft Corporation and contributors.
 # Licensed under the MIT License.
 
-from .latent_position_test import LatentPositionTest
-from .latent_distribution_test import LatentDistributionTest
+from .latent_position_test import latent_position_test, difference_norm
+from .latent_distribution_test import latent_distribution_test
 
-__all__ = ["LatentPositionTest", "LatentDistributionTest"]
+__all__ = ["latent_position_test", "difference_norm", "latent_distribution_test"]
```

Comment:
why is difference norm exposed? also why are the leading underscores removed on it and all other helper functions?

Reply:
difference norm is exposed because the function is being used in the tests for latent position test so without it being exposed the tests can't run successfully

Reply:
you can still import it from the file without adding it to the init.